chrome-extension/content.js

```diff
@@ -120,23 +130,61 @@ function customCreateElement(type, classList, innerHTML = '') {
 const serverUrl = 'http://localhost:8080';
 // TODO: select between dev host and prod host
 
-async function addItem() {
+/**
+ * Returns a Promise that resolves to a <select> element,
+ * a multi-select dropdown menu of category options from fetching db
+ * @param {String} userId 
+ */
+async function getCategoryDropdown(userId) {
+  const categoryDropdown = customCreateElement('select', []);
+  categoryDropdown.setAttribute('multiple', true);
+
+  const defaultOption = customCreateElement('option', [], 'Option: Select a Category');
+  defaultOption.value = '';
+  defaultOption.disabled = true;
+  defaultOption.selected = true; 
+  categoryDropdown.options.add(defaultOption);
+
+  try {
+    const response = await fetch(serverUrl + '/category/getCategories/' + userId);
+    const categories = await response.json();
+    categories.map((category) => {
+      const newOption = customCreateElement('option', [], category.title);
+      newOption.value = category._id;
+      categoryDropdown.options.add(newOption);
+    });
+  } catch (error) {
+    // TODO: Handle this error? 
```

Comment:
For now you can maybe have the drop down disable itself and say something like "unavailable"?

Reply:
To be addressed in later PR- will allow adding new category